hr_infraction/models/hr_infraction.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+# © 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# © 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class HrInfractionCatagory(models.Model):
+    _name = 'hr.infraction.category'
+    _description = 'Infraction Type'
+
+    name = fields.Char(
+        string='Name',
+        required=True,
+        translate=True,
+    )
+    code = fields.Char(
```

Comment:
I don't see where good is being used in other parts of the code or it's application in the real domain so maybe we might want to drop